packages/core/src/command/consolidated/consolidated-create.ts

```diff
@@ -62,6 +65,17 @@ export async function createConsolidatedFromConversions({
   const deduped = deduplicate({ cxId, patientId, bundle: withDups });
   log(`...done, from ${withDups.entry?.length} to ${deduped.entry?.length} resources`);
 
+  const isAiBriefFeatureFlagEnabled = await isAiBriefFeatureFlagEnabledForCx(cxId);
```

Comment:
Can we move this to the `Promise.all` above (line 51 👽 ), so we minimize the processing time, please?